src/Microsoft.AspNetCore.SignalR/IHubClients.cs

```diff
@@ -1,12 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.Collections.Generic;
+
 namespace Microsoft.AspNetCore.SignalR
 {
     public interface IHubClients
     {
         IClientProxy All { get; }
 
+        IClientProxy AllExcept(List<string> excludedIds);
```

Comment:
I thought about `IEnumerable` but there is always a problem with lazy evaluation. `IReadOnlyList` is a good option. Agree that `params` is a little weird (you don't know connectionIds upfront but you need to know the count) but it saves you a few keystrokes if you want to exclude just one connectionId you store in a variable (e.g. `.Others`).